src/test/java/com/auth0/AuthorizeUrlTest.java

```diff
@@ -128,6 +128,19 @@ public void shouldSetStateSameSiteAndNotLegacyCookieWhenConfigured() {
         assertThat(headers, hasItem("com.auth0.state=asdfghjkl; HttpOnly; Max-Age=600; SameSite=None; Secure"));
     }
 
+    @Test
+    public void shouldSetSecureCookieWhenConfigured() {
+        String url = new AuthorizeUrl(client, request, response, "https://redirect.to/me", "code")
+                .withState("asdfghjkl")
+                .withSecureCookie(true)
```

Comment:
I'd add a test with this set to false, checking that when samesite=none this is still being set as secure even if we tell it not to with the setter. (the default behavior)

Reply:
Pushed a commit to add this test case 👍